File: fjt.api/models/RFQAssyQuantityPriceSelectionSetting.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const RFQAssyQuantityPriceSelectionSetting = sequelize.define('RFQAssyQuantityPriceSelectionSetting', {
        id: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER
        },
        qtyID: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        stock: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        price: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        isCheckRequiredQty: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        isLeadTime: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        isDeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        remark: {
            type: Sequelize.STRING,
            allowNull: true
        },
        stockPercentage: {
            type: Sequelize.DECIMAL(16, 6),
            allowNull: true
        },
        packagingID: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        rfqPriceGroupId: {
            allowNull: true,
            type: Sequelize.INTEGER
        },
        rfqPriceGroupDetailId: {
            allowNull: true,
            type: Sequelize.INTEGER
        },
        settingType: {
            allowNull: false,
            type: Sequelize.STRING
        },
        leadTime: {
            allowNull: true,
            type: Sequelize.DECIMAL(10, 1)
        }
    },
        {
            tableName: 'rfq_assy_quantity_price_selection_setting',
            paranoid: true
        });

    RFQAssyQuantityPriceSelectionSetting.associate = (models) => {
        RFQAssyQuantityPriceSelectionSetting.belongsTo(models.RFQAssyQuantity, {
            foreignKey: 'qtyID',
            as: 'rfqAssyQuantity'
        });
        RFQAssyQuantityPriceSelectionSetting.belongsTo(models.ComponentPackagingMst, {
            foreignKey: 'packagingID',
            as: 'componentPackagingMst'
        });
        RFQAssyQuantityPriceSelectionSetting.belongsTo(models.RFQPriceGroup, {
            as: 'rfqPriceGroup',
            foreignKey: 'rfqPriceGroupId'
        });
        RFQAssyQuantityPriceSelectionSetting.belongsTo(models.RFQPriceGroupDetail, {
            as: 'rfqPriceGroupDetail',
            foreignKey: 'rfqPriceGroupDetailId'
        });
    };

    return RFQAssyQuantityPriceSelectionSetting;
};

File: fjt.pricingservice/MySqlDBModel/rfq_assy_quantity_price_selection_setting.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace fjt.pricingservice.MySqlDBModel
{
    [Table("rfq_assy_quantity_price_selection_setting")]
   public class rfq_assy_quantity_price_selection_setting
    {   
        [Key]
        public int id { get; set; }
        public int? qtyID { get; set; }
        public int? stock { get; set; }
        public int? price { get; set; }
        public bool? isCheckRequiredQty { get; set; }
        public bool? isLeadTime { get; set; }
        public bool isDeleted { get; set; }
        [StringLength(255)]
        public string createdBy { get; set; }
        public DateTime createdAt { get; set; }
        [StringLength(255)]
        public string updatedBy { get; set; }
        public DateTime updatedAt { get; set; }
        [StringLength(255)]
        public string deletedBy { get; set; }
        public DateTime? deletedAt { get; set; }
        public string remark { get; set; }
        public virtual rfq_assy_quantity rfq_assy_quantity { get; set; }
        public decimal? stockPercentage { get; set; }
        public int? packagingID { get; set; }
        public string packaging { get; set; }
    }
}
